### PR TITLE
#462: DyTxn.java:60-60: increment should use actions fetched...

### PR DESCRIPTION
**Pull request by pandreev**
please review

### CODE REVIEW

**Comment by davvd** [comment]
Thanks, I'll ask someone to do a code review here, thanks for your pull request

**Comment by davvd** [comment]
@yegor256 the task is yours plz proceed

File: src/main/java/com/thindeck/dynamo/DyTxn.java [hunk 1]
```diff
@@ -51,18 +57,25 @@
...
      * Constructor.
+     *
      * @param itm Item
      */
     public DyTxn(@NotNull final Item itm) {
-        throw new UnsupportedOperationException(itm.toString());
+        this.item = itm;
     }
 
     // @todo #372:30min increment should use actions fetched from item.
```
Comment (yegor256) [inline src/main/java/com/thindeck/dynamo/DyTxn.java]: remove this comment

**Comment by yegor256** [comment]
@pandreev what about a unit test? how do you know that your code really works? besides that, don't forget to run `mvn clean install -Pqulice`

**Comment by yegor256** [comment]
@pandreev we won't merge this pull request, it's too old and there are too many changes in the source code since then, thanks for it anyway

**Comment by davvd** [comment]
@yegor256 Thanks a lot, I just topped your account for **17 mins**, transaction ID `000-3f7ac482`, total time was 4886 hours and 54 mins... review comments (c=2) added as a bonus... +17 added to your rating, at the moment it is: [+10634](http://www.netbout.com/b/35008?open=rating)

**Pull request closed without merging**